### PR TITLE
Template update and a change to out of hours links

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: Safety/out-of-hours-support.rst [hunk 1]
```diff
@@ -39,17 +39,19 @@ Once enrolled, you can `download`_ a desktop application or browser extension to
     Your number will also be accessible to everyone with permissions to the vault.
     If you volunteer, please provide your availability to the best of your knowledge, including holdiays and weekends.
```
Comment (isotuela) [inline Safety/out-of-hours-support.rst:40]: Typo in holidays. 
Also, you mention pingraham in the first paragraph as the person to request access to the Vault, shouldn't his name be removed?

File: Safety/out-of-hours-support.rst [hunk 2]
```diff
@@ -39,17 +39,19 @@ Once enrolled, you can `download`_ a desktop application or browser extension to
...
-    - `#auxtel-operations`_
-    - `#auxtel-virtual-control-room`_
-    - `#rubin-observing`_
+    - `#summit-auxtel`_
+    - `#summit-maintel`_
+    - `#summit-control-room`_
+
+Find :ref:`more information <Daytime-Nighttime-Slack-Channel-Communication-with-Summit-Personnel>` about Slack communications and the different channels being monitored.
 
 - If no response is received from Slack, and summit staff have run out of ideas or need assistance, consult the Emergency Call List.
```
Comment (isotuela) [inline Safety/out-of-hours-support.rst:57]: Can we add the link to the Confluence page where the list of volunteers is? 
There's also a typo on the note: "exhAUst the call list"

File: project/Templates/informative-document.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Include links or further resources for additional follow-up.  
+
+.. warning::
+    Keep this in mind when referring to this information. 
+
+.. _Informative-Document-Main-Information-Section-Subsection:
+
+Subsection
+==========
+
```
Comment (isotuela) [inline project/Templates/informative-document.rst:55]: Maybe add here two Subsection 1 and Subsection2 to show the general structure?

File: project/Templates/procedure-template.rst [hunk 4]
```diff
@@ -36,21 +36,21 @@ Overview
...
+.. _Title-of-Procedure-Precondition:
 
-Prerequisites
-^^^^^^^^^^^^^
+Precondition
+^^^^^^^^^^^^
 
 .. This section should provide simple overview of prerequisites before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
 .. It is preferred to include them as a bulleted or enumerated list.
 .. Do not include actions in this section. Any action by the user should be included at the beginning of the Procedure section below. For example: Do not include "Notify specified SLACK channel. Confirmation is not required." Instead, include this statement as the first step of the procedure, and include "Notification to specified SLACK channel." in the Prerequisites section.
```
Comment (isotuela) [inline project/Templates/procedure-template.rst:46]: I find this sentence very confusing, please consider removing it or rephrasing it.

File: project/Templates/procedure-software-template.rst [hunk 5]
```diff
@@ -1,6 +1,6 @@
-##################
-Title of procedure
-##################
+###########################
```
Comment (isotuela) [inline project/Templates/procedure-software-template.rst:1]: This template doesn't have the form of the others with the labels, explanations and warnings. What kind of operational procedures will be included in the "procedure-software"? I see most or all of them under the "Procedure-template" below.

File: project/Templates/troubleshooting-template.rst [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+.. note::
+    This is a procedure template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Title-of-Troubleshooting-Procedure-Overview:
+
+Overview
+^^^^^^^^
+
+.. In one or two sentances, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure.
```
Comment (isotuela) [inline project/Templates/troubleshooting-template.rst:35]: sentEnces

File: project/Templates/troubleshooting-template.rst [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+Overview
+^^^^^^^^
+
+.. In one or two sentances, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This is a template for a troubleshooting procedure. It is performed by authorized and trained users.
+
+.. _Title-of-Troubleshooting-Procedure-Known-Causes:
+
+Known causes
```
Comment (isotuela) [inline project/Templates/troubleshooting-template.rst:41]: Please consider "Problem Diagnosis" as title here instead of "Known causes"?

**Review by isotuela — CHANGES_REQUESTED** [review]
Looks good! Thank you! 
I have added some comments and minor suggestions for you to consider.

File: Safety/out-of-hours-support.rst (hunk 1 above)
Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:40]: Yup, damn. I will put in A2, I think. 👍

File: Safety/out-of-hours-support.rst (hunk 2 above)
Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:57]: We can't link to Confluence pages, but we can update when this is more official. 👍

File: project/Templates/informative-document.rst (hunk 3 above)
Comment (Slimleashma) [reply project/Templates/informative-document.rst:55]: 👍

File: project/Templates/procedure-template.rst (hunk 4 above)
Comment (Slimleashma) [reply project/Templates/procedure-template.rst:46]: 👍  I removed. It wasn't necessary to put so many constraints.

File: project/Templates/procedure-software-template.rst (hunk 5 above)
Comment (Slimleashma) [reply project/Templates/procedure-software-template.rst:1]: 👍 Removed. It was a leftover from previous templates.

File: project/Templates/troubleshooting-template.rst (hunk 6 above)
Comment (Slimleashma) [reply project/Templates/troubleshooting-template.rst:35]: 👍

File: project/Templates/troubleshooting-template.rst (hunk 7 above)
Comment (Slimleashma) [reply project/Templates/troubleshooting-template.rst:41]: 👍

**Review by isotuela — APPROVED** [review]
Thanks for the changes!

**Pull request merged**